README.md

```diff
@@ -127,21 +129,22 @@ These steps are completed by the editors of the blog. As editor, you might ask q
    
    b. Merge the PR into `staging`.
    
-4. Request a build  of the [staging openliberty.io site from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches) (type `staging` in the **Branch** field of the dialog).
+4. IBM Cloud will automatically rebuild [blogs-staging site](https://blogs-staging-openlibertyio.mybluemix.net/blog/) and [staging site](https://staging-openlibertyio.mybluemix.net/blog/). If you have access, you can track the progress in the [Slack channel](https://app.slack.com/client/T15GKHBT4/C01GXGW1DGQ).  
+~~Request a build of the [staging openliberty.io site from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches) (type `staging` in the **Branch** field of the dialog).~~
 
-5. When the build has finished, check to make sure the blog with its blog tags render correctly on the [staging site](https://staging-openlibertyio.mybluemix.net/blog/). 
+5. When the build has finished, check to make sure the blog with its blog tags render correctly on the [blogs-staging site](https://blogs-staging-openlibertyio.mybluemix.net/blog/) or [staging site](https://staging-openlibertyio.mybluemix.net/blog/). The latter includes the entire site, while the former just has the blog content.  If you need to verify links to other parts of the site (outside of the /blogs/ content) then you'll need to wait for the full [staging site](https://staging-openlibertyio.mybluemix.net/blog/) to build.  
 
    This is the final check before the post is published live on the [production site](https://openliberty.io/blog/).
 
-   If there are any problems found on the [staging site](https://staging-openlibertyio.mybluemix.net/blog/), you must resolve them quickly or revert the PR (the post must not stay in `staging` longer than a couple of hours or you risk someone accidentally publishing it for you).
+   If there are any problems found on with the content in the `staging` branch, you must resolve them quickly or revert the PR (the post must not stay in `staging` longer than a couple of hours or you risk someone accidentally publishing it for you).
```

Comment:
If there are any problems found on with the content in the `staging` branch
-->
If there are any problems with the content in the `staging` branch